src/SDK/Language/Dart.php

```diff
@@ -101,7 +101,8 @@ public function getKeywords()
             "set",
             "yield",
             "required",
-            "default"
+            "extension",
```

Comment:
`default` should also be added here, it's still a keyword?

Reply:
@lohanidamodar It's already added at line `91`, I removed the duplicated one.